llama.cpp

```diff
@@ -1496,6 +1496,9 @@ llama_token llama_token_bos() {
 llama_token llama_token_eos() {
     return 2;
 }
+llama_token llama_token_newline() {
+    return 13;
+}
```

Comment:
The functions for bos/eos are necessary because these are special tokens, you can use `llama_tokenize` instead.

Reply:
Thank you for the heads up. Everything is now in main.cpp. Is this okay now or can it be done better still?